inc/migration.class.php

```diff
@@ -73,7 +73,7 @@ class Migration {
    function __construct($ver) {
 
       $this->deb = time();
-      $this->setVersion($ver);
+      $this->version = $ver;
```

Comment:
For information, this change prevents the call of `addNewMessageArea` triggered in `setVersion` method (for construct case).
`setVersion` is called later in `update_xx_to_yy.php` files

Without that, we have this output:
```html
<div id="migration_message_9.2.2"><p class="center">
   Work in progress | Task Completed
</p></div>
<h3>Update to x.x.x</h3>
```

With the change, the `h3` and `div#migration` tags are in correct order.